fizzbuzz.py

```diff
@@ -1,5 +1,19 @@
 # -*- coding: utf-8 -*-
 
 
-def fizzbuzz():
-    pass
+def fizzbuzz(limit=100):
+    res = []
+    for i in range(1, limit + 1):
```

Comment:
It might be good to split responsibilities into different components/functions/whatever, so that the responsibility to solve the 100 numbers would be different to the responsibility to decide which answer to be given.